HARSHAL/javascript/assignment18/assignment18Q2.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>que2</title>
+        <style>
+            body {
+                text-align: center;
+            }
+            button, label {
+                display: block;
+                width: 20%;
+                margin: auto;
+                margin-top: 20px;
+            } 
+        </style>
+    </head>
+    <body>
+        <fieldset>
+            <label for="number">ENTER NAME: </label>
+            <input id="name" type="name" />
+            <label for="number">ENTER AGE: </label>
+            <input id="age" type="number" />
+            <button onclick="check()">SHOW</button>
+        </fieldset>
+        <script>
+            function check() {
+                var name;
+                var age;
+                var check;
+                name = document.getElementById("name").value;
+                age = document.getElementById("age").value;
+                if (age > 18) {
```

Comment:
Use Try catch (Concept) for throwing error for eligible or not.

Reply:
This is also ok